test/common-utils/scenarios.json

```diff
@@ -41,6 +41,34 @@
             "common-utils": {}
         }
     },
+    "alma-8": {
+        "image": "almalinux:8",
+        "remoteUser": "devcontainer",
+        "features": {
+            "common-utils": {}
+        }
+    },
+    "alma-9": {
```

Comment:
Same comment here!

Reply:
Yeah I noticed. I just pushed the missing .sh files.

Reply:
Commit 1a425bd fixes the missing .sh test scripts.